templates/injector-psp-rolebinding.yaml

```
@@ -0,0 +1,18 @@
+{{- if and (eq (.Values.injector.enabled | toString) "true" ) (and (eq (.Values.global.enabled | toString) "true") (eq (.Values.global.pspEnable | toString) "true") ) }}
+apiVersion: rbac.authorization.k8s.io/v1
+kind: RoleBinding
+metadata:
+  name: {{ template "vault.fullname" . }}-agent-injector-psp
+  namespace: {{ .Release.Namespace }}
+  labels:
+    app.kubernetes.io/name: {{ include "vault.name" . }}
+    app.kubernetes.io/instance: {{ .Release.Name }}
+    app.kubernetes.io/managed-by: {{ .Release.Service }}
+roleRef:
+  kind: ClusterRole
+  name: {{ template "vault.fullname" . }}-psp
```

Should this be binding to the injector psp Role? Otherwise I don't think it's going to pick up the podsecuritypolicies rule.
```suggestion
  kind: Role
  name: {{ template "vault.fullname" . }}-agent-injector-psp
```